SurveyManagementSample/src/main/java/com/verint/xm/smSample/MainActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.verint.xm.smSample;
+
+
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import android.view.View;
+
+import com.verint.xm.sdk.Core;
+import com.verint.xm.sdk.Predictive;
+import com.verint.xm.sdk.SurveyManagement;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+
+    public void onCheckEligibilityClicked(View view) {
+
+        // Launch an invite as a demo
+        SurveyManagement.Companion.checkIfEligibleForSurvey();
+    }
+
+    public void onIncrementSigEventClicked(View view) {
+        SurveyManagement.Companion.incrementSignificantEventCountWithKey("In-session");
```

Comment:
Because this method is marked with `@JvmStatic` in our code, you don't need the `Companion`, i.e.
```suggestion
        SurveyManagement.incrementSignificantEventCountWithKey("In-session");
```